src/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script src="./herbivoresAndCarnivores.js`"></script>
```

Comment:
It seems like there is a typo in the script source path. The backtick (`) at the end of the file path should be removed.